middleware/cors.go

```diff
@@ -7,8 +7,14 @@ import (
 )
 
 var (
+	// TODO: Use OriginValidator
 	allowedOrigins = []string{
-		"http://localhost:*",
+		"http://localhost:3000",
+		"https://localhost:3000",
+		"http://localhost:5000",
+		"https://localhost:5000",
+		"http://localhost:8080",
+		"https://localhost:8080",
```

Comment:
Did you forget `http://localhost:8000`? It's definitely in the top 10 of most used localhost ports. I even checked whatisthemostusedportsforlocalwebdevelopment.supervigtigjournalist.nu